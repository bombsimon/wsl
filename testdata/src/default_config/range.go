package testpkg

func fn1() {
	one := []int{}
	two := []int{} // want "missing whitespace decreases readability"
	for range two {
		panic(err)
	}
}

func fn2() {
	two := []int{}
	one := []int{}
	for range two { // want "missing whitespace decreases readability"
		panic(err)
	}
}

func fn3() {
	for range make([]int{}, 0) {
		panic(foo)
	}
	for range make([]int{}, 0) { // want "missing whitespace decreases readability"
		panic(bar)
	}
}
