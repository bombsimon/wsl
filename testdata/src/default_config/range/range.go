package testpkg

func fn1() {
	a := []int{}
	b := []int{} // want `missing whitespace above this line \(too many statements above range\)`
	for range b {
		panic(1)
	}

	_ = a
	_ = b
}

func fn2() {
	b := []int{}
	a := []int{}
	for range b { // want `missing whitespace above this line \(no shared variables above range\)`
		panic(1)
	}

	_ = a
	_ = b
}

func fn3() {
	for range make([]int, 0) {
		panic("")
	}
	for range make([]int, 0) { // want `missing whitespace above this line \(invalid statement above range\)`
		panic("")
	}
}
