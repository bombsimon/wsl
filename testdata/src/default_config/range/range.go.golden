package testpkg

func fn1() {
	a := []int{}

	b := []int{} // want "missing whitespace decreases readability"
	for range b {
		panic(a)
	}
}

func fn2() {
	b := []int{}
	a := []int{}

	for range b { // want "missing whitespace decreases readability"
		panic(a)
	}
}

func fn3() {
	for range make([]int, 0) {
		panic("")
	}

	for range make([]int, 0) { // want "missing whitespace decreases readability"
		panic("")
	}
}
